src/simple_cache_client.rs

```diff
@@ -1917,35 +1918,7 @@ impl SimpleCacheClient {
         limit: impl Into<Option<NonZeroU32>>,
         range: Option<sorted_set::Range>,
     ) -> MomentoResult<MomentoSortedSetFetchResponse> {
-        use momento_protos::cache_client::sorted_set_fetch_request::{All, Limit, NumResults};
-        use momento_protos::cache_client::sorted_set_fetch_response::SortedSet;
-
-        let num_results = limit
-            .into()
-            .map(|v| NumResults::Limit(Limit { limit: v.into() }))
-            .unwrap_or_else(|| NumResults::All(All {}));
-
-        let request = self.prep_request(
-            cache_name,
-            SortedSetFetchRequest {
-                set_name: set_name.into_bytes(),
-                order: order.into(),
-                num_results: Some(num_results),
-                range,
-            },
-        )?;
-
-        let response = self
-            .data_client
-            .sorted_set_fetch(request)
-            .await?
-            .into_inner();
-        Ok(MomentoSortedSetFetchResponse {
-            value: response.sorted_set.and_then(|s| match s {
-                SortedSet::Found(found) => Some(found.elements),
-                SortedSet::Missing(_) => None,
-            }),
-        })
+        todo!();
```

Comment:
this api is getting overhauled. When I updated the client protos, it was breaking this api. Since this api is still getting hashed out, I decided to just leave it as a `todo()`

Reply:
oh, I see.  The API is settled now.  The support for the final API has already been added to the node.js SDK.

Reply:
who originally added SortedSet to this SDK?  we need to check and see if it is being used by someone.  If not, maybe this is okay for now.  cc: @kvcache

Reply:
I do not know whether this is consumed, but @brayniac is the most likely person I am aware of to be using it.

Is there an issue we can link here instead like
```
todo!("This api was reworked and is pending implementation in the rust sdk: https://github.com/foo/bar");
```
instead of a contextless panic?